website/content/middleware/jwt.md

```diff
@@ -27,50 +27,92 @@ e.Use(middleware.JWTWithConfig(middleware.JWTConfig{
 }))
 ```
 
+*Usage with different JWT library/repository*
+
+> NB: Due backwards compatibility reasons Echo still uses unmaintained, with potential security risks `dgrijalva/jwt-go` version. We recommend using maintained fork ala https://github.com/golang-jwt/jwt
+
+```go
+// Use maintained fork (https://github.com/golang-jwt/jwt) of `dgrijalva/jwt-go` instead
+// At this moment: go get github.com/golang-jwt/jwt@v3.2.1
+import (
+  "github.com/golang-jwt/jwt"
+)
+```
+
+To use different JWT library you need to implement custom `JWTConfig.ParseTokenFunc` function:
```

Comment:
To use a different JWT library you need to implement a custom `JWTConfig.ParseTokenFunc` function